addons/sale/models/res_partner.py

```diff
@@ -68,6 +68,7 @@ def action_view_sale_order(self):
         action = self.env['ir.actions.act_window']._for_xml_id('sale.act_res_partner_2_sale_order')
         all_child = self.with_context(active_test=False).search([('id', 'child_of', self.ids)])
         action["domain"] = [("partner_id", "in", all_child.ids)]
+        action["context"] = {'search_default_partner_id': all_child.ids}
```

Comment:
Same as above and elsewhere, if the filter overlaps with the forced domain it will probably feel a bit weird

Reply:
yes, I will make sure it doesn't :+1: